### PR TITLE
SearchUser changed to styled-components

### PR DESCRIPTION
**Pull request by geoseong**
I don't know how to make 100% of jest's `Functional` test.

I think this is related to `styled-component`, but I haven't figured it out...

### CODE REVIEW

File: src/components/screen/SearchUser.js [hunk 1]
```diff
@@ -25,35 +26,80 @@ import { ProfileModalConsumer } from '../../providers/ProfileModalProvider';
...
+import styled from 'styled-components/native';
 
-const styles: Styles = StyleSheet.create({
-  container: {
-    flex: 1,
-    backgroundColor: 'transparent',
-    flexDirection: 'column',
-    alignItems: 'center',
-    justifyContent: 'center',
+const FriendsSampleData = [
```
Comment (hyochan) [inline src/components/screen/SearchUser.js]: It's not `FriendsSampleData`, it should be `UserSampleData`.
Also, I think it's not a good idea to use `upper camel case` with a variable (not a class, type or a render component).
If you really want it to be capital, you better name it like `USER_SAMPLE_DATA`. I recommend this to be `userSampleData`.

File: src/components/screen/SearchUser.js [hunk 2]
```diff
@@ -25,35 +26,80 @@ import { ProfileModalConsumer } from '../../providers/ProfileModalProvider';
...
-    top: 18,
+  {
+    uid: '8', displayName: 'geoseong', photoURL: IC_BACK, statusMsg: 'healthy', isOnline: '', friends: '', Chatrooms: '', created: '', updated: '',
   },
-});
+  {
+    uid: '9', displayName: 'hyochan', photoURL: IC_ICON, statusMsg: 'healthy', isOnline: '', friends: '', Chatrooms: '', created: '', updated: '',
+  },
+];
 const AnimatedFlatList = Animated.createAnimatedComponent(FlatList);
```
Comment (hyochan) [inline src/components/screen/SearchUser.js]: same with above.

File: src/components/screen/SearchUser.js [hunk 3]
```diff
@@ -179,77 +161,45 @@ class Screen extends Component<Props, State> {
...
                       translateY: this.scrollY.interpolate({
-                        inputRange: [-50, 0, 50, 100],
-                        outputRange: [0, 0, -50, -50],
+                        inputRange: [0, 50, 100],
+                        outputRange: [0, 25, 0],
                       })
-                    }],
+                    }]
                   }}
                   contentContainerStyle={this.containerStyle}
```
Comment (hyochan) [inline src/components/screen/SearchUser.js]: name the render stuff like `this.renderContainer()`. Naming looks bit ambiguous.

File: src/components/screen/__tests__/SearchUser.test.js [hunk 4]
```diff
@@ -7,6 +7,8 @@ import { IC_BACK, IC_SEARCH, IC_ICON } from '../../../utils/Icons';
 import renderer from 'react-test-renderer';
 import { shallow, render } from 'enzyme';
 
+jest.useFakeTimers(); // related to Animated Component
```
Comment (hyochan) [inline src/components/screen/__tests__/SearchUser.test.js]: I hope you to use this inside `describe` or `it` statement (where you need it).
I think stating it as global isn't a good idea.

**Comment by hyochan** [comment]
@geoseong Refer to test coverage, It's ok. You shouldn't have to be perfect. I've written review mostly on variable naming.

P.S. We are even planning to switch to `functional component` with `react-hook`, so our test codes may be renewed.

File: src/components/screen/__tests__/SearchUser.test.js (hunk 4 above)
Comment (geoseong) [reply src/components/screen/__tests__/SearchUser.test.js]: I moved this code to place where in the `describe`

File: src/components/screen/SearchUser.js (hunk 3 above)
Comment (geoseong) [reply src/components/screen/SearchUser.js]: got it 😅

File: src/components/screen/SearchUser.js (hunk 1 above)
Comment (geoseong) [reply src/components/screen/SearchUser.js]: okay. I renamed it

File: src/components/screen/SearchUser.js [hunk 5]
```diff
@@ -190,7 +190,7 @@ class Screen extends Component<Props, State> {
                       })
                     }]
                   }}
-                  contentContainerStyle={this.containerStyle}
+                  contentContainerStyle={this.renderContainer}
```
Comment (hyochan) [inline src/components/screen/SearchUser.js]: Oops sorry. This isn't a `render` function. This should be reverted. Sorry.

File: src/components/screen/__tests__/SearchUser.test.js [hunk 6]
```diff
@@ -76,18 +75,18 @@ describe('[serachUser] interaction', () => {
       expect(outer.instance().state.users[0].displayName).toEqual(inputData.displayName);
 
       outer.instance().onSearch('');
-      expect(outer.instance().containerStyle()).toEqual(null);
+      expect(outer.instance().renderContainer()).toEqual(null);
```
Comment (hyochan) [inline src/components/screen/__tests__/SearchUser.test.js]: Same as above sorry for this.

**Review by hyochan — APPROVED** [review]
👍

**Pull request merged**